### ISSUE TITLE
include feature to only consider certain tags

### ISSUE THREAD

**Issue by Emilgardis** [snok/container-retention-policy#10]
I want a feature to consider the removal only for certain tags which match a string (with regex/wildcard)

I imagine it would be something like this:

```
- uses: snok/container-retention-policy@v1
  with:
    image-names: dev, web, test
    cut-off: two hours ago UTC+2
    timestamp-to-use: updated_at
    account-type: org
    org-name: google
    keep-at-least: 1
    token: ${{ secrets.PAT }}
    consider-tags: sha-*
```

this would allow the user to only delete old images which match this/these filters, and not anything else. 

bikeshed on the name `consider-tags`, maybe `include-tags`, `filter-tags`?

This would also have to then include a feature to specify if untagged images should be considered as well.

Slightly related to #9

**Comment by sondrelg** [comment]
So where `skip-tags` opts-out, you'd want something like `include-tags` to define the set?

If we have images:

```
web:latest-3.10
web:latest-3.10-1
web:latest-3.10-2
web:latest-3.10-3
web:latest-3.9
web:latest-3.9-1
web:latest-3.9-2
web:latest-3.9-3
```

Then with 

### Scenario 1

```
skip-tags: None
include-tags: None 
``` 

Then the job should delete all unused  images.

### Scenario 2

```
skip-tags: 3.9
include-tags: None 
``` 

Then the job should delete all images except just `web:latest-3.9`


### Scenario 2

```
skip-tags: 3.9-*  # assuming #9 is also fixed
include-tags: None 
``` 

Then the job should delete all 3.10 images.

### Scenario 3

```
skip-tags: 3.9-1
include-tags: 3.9-*
``` 

Then the job should delete all 3.9 images except `latest-3.9-1`

--------------

In other words, this new feature would filter down the set of images, then the skip-tags feature would filter out images afterwards?

Am I missing anything? 🙂

**Comment by Emilgardis** [comment]
That's pretty much it 👍
